e2e/testcases/cli_test.go

```diff
@@ -833,3 +848,235 @@ func TestNomosVetNamespaceRepo(t *testing.T) {
 		})
 	}
 }
+
+func TestCLIBugreportNomosRunningCorrectly(t *testing.T) {
+	nt := nomostest.New(t, nomostesting.NomosCLI, ntopts.RequireGKE(t))
```

Comment:
`ntopts.RequireGKE(t)`: is this required?

Reply:
The test errors out when running with kind cluster, that has trouble exporting the kubeconfig or dump the log. I got it running in GKE clusters.

Reply:
Can the old bat test run on kind clusters?

Reply:
It actually could.. 

```
-  # Fixes a problem when running the tests locally
-  if [[ -z "${KUBECONFIG+x}" ]]; then
-    export KUBECONFIG="${HOME}/.kube/config"
-  fi
```
This seemed to be doing the trick but looks confusing

Reply:
Fixed, it's because of the resource validation predicate. Now the test works with kind.